src/main/resources/google_checks.xml

```diff
@@ -48,6 +48,11 @@
 
     <module name="TreeWalker">
         <module name="OuterTypeFilename"/>
+        <module name="WhitespaceAfter">
+            <property name="tokens"
+            value="COMMA, SEMI, TYPECAST, LITERAL_IF, LITERAL_ELSE,
+            LITERAL_WHILE, LITERAL_DO, LITERAL_FOR, DO_WHILE"/>
```

Comment:
Please indent LITERAL_WHILE to be aligned with COMMA above it.

Reply:
Done